### ISSUE TITLE
Should we change the Healthy LifeStyle link?

### ISSUE THREAD

**Issue by galexopoulos** [zkotti/fun-in-lockdown#18]
In the Healthy LifeStyle link should we keep it as existent, or create a separate link for each of the 3 subcategories?

Current: 
* [Healthy Lifestyle](#health) ([Diet](#diet), [Workout](#workout), [Sleep Quality](#sleep_quality))

Modified: 
* ([Diet](#diet)
* [Healthy Lifestyle](#health)
* [Sleep Quality](#sleep_quality))
* [Workout](#workout)

**Comment by zkotti** [comment]
Indeed, [Workout](https://github.com/zkotti/fun-in-lockdown/blob/main/guide.md#workout) could be merged into the [Exercise](https://github.com/zkotti/fun-in-lockdown/blob/main/guide.md#exercise) category, amending any duplicate content. Feel free to change it if you like.
